internal/docstore/mongodocstore/mongo.go

```diff
@@ -289,49 +290,59 @@ func (c *collection) toMongoFieldPath(fp []string) string {
 	return strings.Join(fp, ".")
 }
 
-func (c *collection) create(ctx context.Context, a *driver.Action) error {
-	// See https://docs.mongodb.com/manual/reference/method/db.collection.insertOne
-	id, err := c.docID(a.Doc)
-	// If the user provides a function to get the ID rather than a field, then Create
-	// can't make a unique ID for a document that's missing one, because it doesn't
-	// know how to create it or where to store it in the original document.
-	if err != nil && c.idField == "" {
-		return err
+func sliceToLower(s []string) {
+	for i, e := range s {
+		s[i] = strings.ToLower(e)
 	}
-	mdoc, err := c.encodeDoc(a.Doc, id)
+}
+
+func (c *collection) create(ctx context.Context, a *driver.Action) error {
+	// See https://docs.mongodb.com/manual/reference/method/db.collection.insertOne.
```

Comment:
This and L309 might better be summarized and put on top of `create`.

Reply:
Moved this line above the function declaration.